includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1504,4 +1504,18 @@ function ( $token ) {
 			array_values( $tokens )
 		);
 	}
+
+	/**
+	 * Disables gateway.
+	 */
+	public function disable() {
+		$this->update_option( 'enabled', 'no' );
```

Comment:
I didn't know about `WC_Settings_API::update_option`. I'll need to fix some conflicts, but this is way better than the way I'm doing it in my PR :D 👍 👏